### PR TITLE
Fix source retrieval inside frame/iframe

### PR DESCRIPTION
**Pull request by imurchie**
Rather than getting the source of the frameset, should get the source of the frame itself when inside a frame.

### CODE REVIEW

File: lib/commands/source.js [hunk 1]
```diff
@@ -10,8 +10,8 @@ const APPIUM_SRC_XML = '<?xml version="1.0" encoding="UTF-8"?><AppiumAUT/>';
 // is fixed, we need to translate the request
 commands.getPageSource = async function () {
   if (this.isWebContext()) {
-    let cmd = 'document.getElementsByTagName("html")[0].outerHTML';
-    return await this.remote.execute(cmd);
+    const script = 'return document.documentElement.outerHTML';
```
Comment (mykola-mokhnach) [inline lib/commands/source.js]: is _return_ needed there?

Comment (imurchie) [reply lib/commands/source.js]: I don't understand what you mean? There is a return on line 14.

Comment (mykola-mokhnach) [reply lib/commands/source.js]: I mean the previous cmd had not return : `'document.getElementsByTagName("html")[0].outerHTML'`

Comment (imurchie) [reply lib/commands/source.js]: Oh! Haha. Yes, that is necessary because we are running the execute_script atom. The old execute worked differently.

**Pull request merged**